app/models/rsvp.rb

```
@@ -0,0 +1,15 @@
+class Rsvp < ApplicationRecord
+  STATUSES = %w[yes maybe no]
```

RuboCop (1.75.5)

[convention] Freeze mutable objects assigned to constants.

(Style/MutableConstant)